nf_amf/Dockerfile

```diff
@@ -31,3 +30,5 @@ VOLUME [ "/free5gc/support/TLS" ]
 
 # Exposed ports
 EXPOSE 29518
+
+CMD [ "./amf", "-amfcfg", "../config/amfcfg.conf" ]
```

Comment:
if the command put in dockerfile, I think we should not point the config file.

The config file position will change dependent on how you mount in the compose file, so I think config file path assignment must put into compose file?

Reply:
But, giving a second thought, when `command` is specified in docker-compose, it will override the default one (of the dockerfile) as explained here: https://docs.docker.com/compose/compose-file/#command

We can say then that if we change the mount position we can always add a new command to point to the correct new position.

What do you think ?

Reply:
How about copy an default config which put on the default path and bind the address to `0.0.0.0`. Then set default CMD to for example `CMD ["./amf"].

And then in compose file, we sill mount another config file that will bind the ip using alias address. the `command` or `entrypoint` will be set the config parameter to the mounted config file?

Reply:
You mean we make an entrypoint like `ENTRYPOINT [ "./amf" ]` and let the `command` in compose file provide config files. It will work except for the `n3iwf` because we need to run some `ip` configs before running the `n3iwf`, so we can't use `./n3iwf` as an entrypoint. 

You know what, let's keep the old way using `command` in compose file. It seems the simplest way, thus the best. I like keeping things simple :+1: